agave_app/AppearanceSettingsWidget.cpp

```diff
@@ -260,17 +262,17 @@ QAppearanceSettingsWidget::createLightingControls(QAction* pLightRotationAction)
   m_lt0gui.m_sizeSlider->setRange(0.1, 5.0);
   m_lt0gui.m_sizeSlider->setSingleStep(5.0 / 100.0);
   m_lt0gui.m_sizeSlider->setValue(1.0);
-  sectionLayout->addRow("AreaLight Size", m_lt0gui.m_sizeSlider);
+  sectionLayout->addRow("Size", m_lt0gui.m_sizeSlider);
   QObject::connect(
     m_lt0gui.m_sizeSlider, &QNumericSlider::valueChanged, this, &QAppearanceSettingsWidget::OnSetAreaLightSize);
 
   m_lt0gui.m_distSlider = new QNumericSlider();
   m_lt0gui.m_distSlider->setStatusTip(tr("Set distance for area light"));
   m_lt0gui.m_distSlider->setToolTip(tr("Set distance for area light"));
-  m_lt0gui.m_distSlider->setRange(0.1, 100.0);
+  m_lt0gui.m_distSlider->setRange(0.1, 10.0);

```

Comment:
Out of curiosity, why did you change this value from 100 to 10?

Reply:
This value is a distance from light source to volume.  The volume size is normalized to 1.0 so putting the light 100 units away is not very useful.  It keeps the slider range more manageable for practical numbers.  The good news is that modifying the slider range does not hurt the ability to load and save values outside this range.